### ISSUE TITLE
input custom time step data

### ISSUE THREAD

**Issue by rheinheimer** [pywr/pywr#792]
With reference to #773, I am trying to input data (max_flow, etc.) as, for example, volume/month instead of volume/day.

As far as I can tell, this is not possible, since when I do this (with the model frequency set to 'M'), Pywr seems to assume the input data is volume/day. As far as I can tell anyway. Is there some way to override this, for example via subclassing?

**Comment by jetuk** [comment]
You are correct. Pywr assumes input is volume/day mean over the period of the data. 

We could add an optional step to `align_and_resample_dataframe` that divided the data by the number of days in the period. This would then be exposed as an optional keyword to the dataframe parameters. Something like:

```json
{
  "url": "mydata.h5",
  "key": "monthly_total_flows",
  "period_totals": true
}
```

Not sure I like the "period_totals" name though.
